src/components/ModalDelete.js

```diff
@@ -0,0 +1,28 @@
+import Modal from "./Modal.js";
+
+export default class ModalDelete extends Modal {
+  constructor({ modalSelector, cardRemovalHandler }, card) {
+    super({ modalSelector });
+    this._deleteButton = document.getElementById("delete-image-confirm");
+    this._cardRemovalHandler = cardRemovalHandler;
+    this.card = card;
+  }
+  open() {
+    this._setEventListeners();
+    super.open();
+    this._deleteButton.innerText = "Yes";
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.